docs/source/index.rst

```diff
@@ -38,16 +41,21 @@ This is the documentation for Mantid |release|.
    :width: 200px
    :align: right
 
-* :ref:`Training`
+* :ref:`training`
```

Comment:
Personal preference here: I think algorithms should be first, since most of the time people are looking for something specific when they start the help. Feel free to ignore.

Reply:
I think we need to wait for the results of Pete's poll to decide this one.